src/script/lua_api/l_storage.cpp

```diff
@@ -110,7 +110,7 @@ void StorageRef::Register(lua_State *L)
 
 	lua_pop(L, 1);  // drop metatable
 
-	luaL_openlib(L, 0, methods, 0);  // fill methodtable
+	luaL_register(L, nullptr, methods);  // fill methodtable
```

Comment:
rebase failure ?

Reply:
These changes appeared during merging, not sure why since merging is supposed to avoid this.
If you merge these will disappear so don't worry.

Reply:
can I resolve this ?